qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -87,6 +111,30 @@ def __init__(
         self._initial_state = initial_state
         self._mixer = mixer
         self._reps = reps
+        self.use_swap_strategies = use_swap_strategies
+        self.use_initial_mapping = use_initial_mapping
+        self.use_pulse_efficient = use_pulse_efficient
+        self._alpha = alpha
```

Comment:
Could you add getters and setters for all of these to be consistent with the other existing properties?